ext/cgi/escape/escape.c

```diff
@@ -1,5 +1,6 @@
 #include "ruby.h"
 #include "ruby/encoding.h"
+#include <stdbool.h>
```

Comment:
Why not avoid using stdbool.h unless there is a special reason? The cgi gem might be compiled with older ruby that does not assume C99.

Reply:
Seemed cleaner to me, but sure I can remove it.

Reply:
Done.